client/tasks.py

```diff
@@ -0,0 +1,49 @@
+from logging import getLogger
+
+from celery.task import task
+from django.conf import settings
+from openedx.features.openedx_nodebb_discussion.client.categories import NodeBBCategory
+from openedx.features.openedx_nodebb_discussion.client.users import NodeBBUser
+
+RETRY_DELAY = 10
+
+log = getLogger(__name__)
+
+
+def handle_response(caller, task_name, job_type, status_code, response, username):
+    """
+    Logs the response of the specific NodeBB API call
+    """
```

Comment:
Add a comment here explaining why we have chosen to retry on certain response codes and not in others.